linera-witty/src/runtime/wasmtime/mod.rs

```diff
@@ -55,11 +55,13 @@ impl<UserData> AsContextMut for EntrypointInstance<UserData> {
 impl<UserData> Instance for EntrypointInstance<UserData> {
     type Runtime = Wasmtime;
     type UserData = UserData;
-    type UserDataReference<'a> = &'a UserData
+    type UserDataReference<'a>
+        = &'a UserData
```

Comment:
Some reformatting. It occurred after a change of compiler version locally. These reformatting occurs for version 1.79 / 1.80 / 1.81

I expected that this would result in an error in the CI. But surprisingly no.

Reply:
Can you undo the unnecessary changes?

Reply:
Ok, undone.

Reply:
are they?

Reply:
Ah, sorry, it did not pass due to the `pre-push` limit. Now it is undone.